data_steward/cdr_cleaner/cleaning_rules/section_participation_concept_suppression.py

```diff
@@ -0,0 +1,131 @@
+"""
+Record suppress (drop the rows) responses related to the section_particiption concept (*_concept_id = 903632)
+in both registered and controlled tiers.
+
+Original Issues: DC-1641
+"""
+
+# Python imports
+import logging
+
+# Project imports
+from cdr_cleaner.cleaning_rules.deid.concept_suppression import AbstractBqLookupTableConceptSuppression
+from constants.cdr_cleaner import clean_cdr as cdr_consts
+from common import JINJA_ENV, OBSERVATION
+from utils import pipeline_logging
+
+# Third party imports
+from google.cloud.exceptions import GoogleCloudError
+
+LOGGER = logging.getLogger(__name__)
+
+SUPPRESSION_RULE_CONCEPT_TABLE = 'section_participation_concepts'
+
+SECTION_PARTICIPATION_CONCEPT_QUERY = JINJA_ENV.from_string("""
+CREATE OR REPLACE TABLE `{{project_id}}.{{sandbox_id}}.{{concept_suppression_lookup_table}}` AS
+SELECT c.*
+FROM `{{project_id}}.{{dataset_id}}.concept` c
+WHERE c.concept_id IN (903632)
+""")
+
+
+class SectionParticipationConceptSuppression(
+        AbstractBqLookupTableConceptSuppression):
```

Comment:
At first I assumed only `observation_source_concept_id` would have the concept_id in observation, after looking it up in Athena. However the ticket specifies all tables so this is the intended approach and reduces work, so this is great.

Reply:
I only have observation in affected_tables. Do you think I should change it to all CDM tables then?

Reply:
I'm not sure, let me ask on the ticket.